src/utils/read-operations.ts

```diff
@@ -14,8 +14,30 @@ interface FilePayload {
   readonly filename: string;
 }
 
-export const readOperationsRepository = (target = join(__dirname, '..', '..', 'data')) =>
-  readFileOrDirectoryRecursively(target) as OperationsRepository;
+export const readOperationsRepository = (target = join(__dirname, '..', '..', 'data')) => {
+  const rawOperations = readFileOrDirectoryRecursively(target) as OperationsRepository;
```

Comment:
I'm trying to understand the rationale behind this change, can you give a brief overview ?

Reply:
Empty folders appear as undefined objects. We ideally need those folders to be empty objects.
This change allows us to read empty folders as empty objects.